frontend/src/components/Comments/Comments.jsx

```diff
@@ -73,9 +74,9 @@ export default function Comments({ ticker }) {
         <div className="comments">
             <h2>Comments</h2>
             <div className="past-comments">
-                {comments?.map((comment) => {
+                {comments?.sort(compareCommentsByDate).map((comment) => {
                     return (
-                        <div className="comment-details" key={comment.content}>
+                        <div className="comment-details" key={`${comment.createdAt}-${comment.content}`}>
```

Comment:
Nit: Is it possible to use the comment database ID here?

Reply:
Good idea